tests/miniwallet/conftest.py

```diff
@@ -9,12 +9,12 @@
 
 @pytest.fixture(scope="package")
 def target_client(diem_client: jsonrpc.Client) -> RestClient:
-    return start_app(diem_client, "target-wallet").create_client()
+    return start_app(diem_client, "targetwallet").create_client()
```

Comment:
I think we probably should keep original name "target-wallet", don't see a reason to rename it.